### DISCUSSION TITLE
go to definition not working

### DISCUSSION THREAD

**Discussion by dvic** [elixir-tools/discussions#2] [Q&A] [answered]
I'm using Astronvim and I noticed that `gd` doesn't work.

If I just install `elixirls` using Mason (and no additional config) then go to definition works (the `g` action brings up a menu and it shows also *Show the definition of current symbol* as an action).

Do I need to configure key bindings manually to make this action work? Or am I missing something else here?

**Comment by dvic** [comment]
By the way, I am using 0.14.6 (see https://github.com/elixir-tools/elixir-tools.nvim/pull/97), could that be the problem?

**Comment by dvic** [comment]
Also, `LspInfo` shows `autostart: false`, is this correct?

**Comment by mhanberg** [comment]
It doesn't configure keybinds for you.

**Reply by dvic** [reply]
Ahh I see, but I'm a bit confused because I think Astro sets this up for me: https://astronvim.com/Basic%20Usage/mappings#lsp-mappings (I also don't see anything in Mason related to this). I'll have to dig more into this.

**Reply by mhanberg** [reply, marked as answer]
Call this function in the on on_attach callback you can pass when calling the elixir tools setup function https://github.com/AstroNvim/AstroNvim/blob/52adcb7b734e0a5f7b2a8916a10ab67939f1dc4d/lua/astronvim/utils/lsp.lua#L128

**Reply by dvic** [reply]
Works like a charm, thanks!
